chapters/synchronous.tex

```diff
@@ -1213,7 +1213,7 @@ \subsection{Solver Methods}\label{solver-methods}
 
 The solvers are defined with respect to the underlying ordinary
 differential equation in state space form to which the continuous-time
-partition can be transformed, at least conceptually ($t$ is time,
+base-clock partition can be transformed, at least conceptually ($t$ is time,
```

Comment:
Time to go sentence-based.

Reply:
By the way, shouldn't this be _sub-clock_?
```suggestion
sub-clock partition can be transformed, at least conceptually ($t$ is time,
```

Reply:
Yes, I was confused by "continuous-time partition"; as we only have "continuous-time base-clock partition" but this is a description of the transformation of the "conceptual non-discretized" version of the _discretized sub-clock partition_.